armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -724,6 +724,7 @@ def getParamWithBlockLevelMax(a, paramName):
                         # this assembly is in the excluded location list. skip it.
                         continue
 
+                # TODO: I'm pretty sure this is broken.
```

Comment:
We added this `TODO`.   What's up with this?

Reply:
I don't remember inspecting this module, but the use of the `zones` tooling itself looks fine to me.

Reply:
Make sense, I probably added the TODO.

I removed it. Thanks!